content/en/docs/concepts/storage/volumes.md

```diff
@@ -1391,72 +963,489 @@ plugin path on each node and in some cases the control plane nodes as well.
 Pods interact with FlexVolume drivers through the `flexvolume` in-tree volume plugin.
 For more details, see the [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md) examples.
 
-## Mount propagation
+### persistentVolumeClaim {#persistentvolumeclaim}
 
-Mount propagation allows for sharing volumes mounted by a container to
-other containers in the same pod, or even to other pods on the same node.
+A `persistentVolumeClaim` volume is used to mount a
+[PersistentVolume](/docs/concepts/storage/persistent-volumes/) into a pod. PersistentVolumeClaims are a way for users to "claim" durable storage (such as a GCE PersistentDisk or an iSCSI volume) without knowing the details of the particular cloud environment.
 
-Mount propagation of a volume is controlled by the `mountPropagation` field
-in `Container.volumeMounts`. Its values are:
+See the information about [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) for more
+details.
 
-* `None` - This volume mount will not receive any subsequent mounts
-  that are mounted to this volume or any of its subdirectories by the host.
-  In similar fashion, no mounts created by the container will be visible on
-  the host. This is the default mode.
+## External Volume Providers
```

Comment:
```suggestion
## External storage providers
```
I'm not sure that I could explain what these are external to.

Reply:
The concept was "externally managed" (i.e., cloud providers, X as a service), but I'm open to alternative terminology.